src/Control/Carrier/Interpret.hs

```diff
@@ -33,6 +33,7 @@ import Control.Monad.Fail as Fail
 import Control.Monad.Fix
 import Control.Monad.IO.Class
 import Control.Monad.Trans.Class
+import Control.Monad.IO.Unlift (MonadUnliftIO)
```

Comment:
Whichever of this PR and #419 gets merged first should make sure to derive an instance for `Choosing` as well.

Reply:
…and should note it in the changelog.